.travis.yml

```diff
@@ -1,35 +1,39 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "4"
 sudo: false
 matrix:
   include:
-    - node_js: "0.10"
+    - node_js: "4"
```

Comment:
no need to necessarily change this but we could just do a bunch of `- env:` instead of setting `node_js` each time